zh/upgrade-tidb-operator.md

```diff
@@ -17,12 +17,23 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
 
 1. 更新 Kubernetes 的 CustomResourceDefinition (CRD)。关于 CRD 的更多信息，请参阅 [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)。
 
-    {{< copyable "shell-regular" >}}
+    * 如果 Kubernetes 版本大于等于 1.16:
 
-    ```shell
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd.yaml && \
-    kubectl get crd tidbclusters.pingcap.com
-    ```
+         {{< copyable "shell-regular" >}}
+
+         ```shell
+         kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd.yaml && \
```

Comment:
The changes will not be in v1.2.

Reply:
the version in URL will be replaced after release v1.3

Reply:
The `master` branch is for `dev` doc, which should match with the `master` branch in TiDB Operator.

Reply:
so, Should we need to change all versions to `master` in the dev doc?
It means that we should replace `master` to the release version when we release a new version.

Reply:
Yes.